lib/internal/http2/compat.js

```diff
@@ -99,8 +99,8 @@ function onStreamData(chunk) {
 function onStreamTrailers(trailers, flags, rawTrailers) {
   const request = this[kRequest];
   if (request !== undefined) {
-    Object.assign(request[kTrailers], trailers);
-    request[kRawTrailers].push(...rawTrailers);
+    request[kTrailers] = trailers;
+    request[kRawTrailers] = rawTrailers;
```

Comment:
This line has a different meaning. If `kRawTrailers` already contained entries, it would now remove those.

Reply:
@BridgeAR: true, but as far as I can see `kRawTrailers` never contains entries...